cosmo_tester/config_schemas/manager_image_names_centos.yaml

```diff
@@ -1,11 +1,17 @@
 namespace: manager_image_names_centos
-master:
+"master":
   description: Name of main manager image to test. This will be used in all manager tests.
-  default: cloudify-manager-premium-5.1.1
-installer:
+  default: cloudify-manager-premium-5.2.0.dev1
+"installer":
   description: Name of image with manager installer for bootstrap or cluster installs.
+  default: cfy-installer-premium-5.2.0.dev1
+"5_1_1_installer":
+  description: Name of image with 5.1.1 manager installer for bootstrap or cluster installs.
   default: cfy-installer-premium-5.1.1
-5_1_0_installer:
+"5_1_1":
+  description: Name of manager image for 5.1.1. This will be used in all manager tests.
```

Comment:
I think "master" is the one that will be used in all manager tests. If yes, then "This will be used in all manager tests." should be deleted.

Reply:
Yes updated